content/terraform-enterprise/v202507-1/docs/enterprise/deploy/reference/configuration.mdx

```diff
@@ -529,8 +529,11 @@ Required when `TFE_OBJECT_STORAGE_TYPE` is `s3`.
 
 #### `TFE_OBJECT_STORAGE_S3_ENDPOINT`
 
-S3 endpoint. Useful when using a private S3 endpoint. Leave blank to use the
-default AWS S3 endpoint. Defaults to `""`.
+S3 endpoint. Useful when using a private S3 endpoint. 
+
+Example: `TFE_OBJECT_STORAGE_S3_ENDPOINT: https://s3.example.com`
```

Comment:
Same comment!

The following example sets your S3 endpoint to a private S3 endpoint named `https://s3.example.com` :

```shell-session
TFE_OBJECT_STORAGE_S3_ENDPOINT: https://s3.example.com
```